UPGRADE-8.1.md

```diff
@@ -0,0 +1,7 @@
+UPGRADE FROM 8.0 to 8.1
+=======================
+
+HttpKernel
+----------
+
+ * Deprecate the `Extension` class, use the `Extension` class from the DependencyInjection component instead
```

Comment:
Let's add the diff of the change to be done.

````suggestion
 * Deprecate the `Symfony\Component\HttpKernel\DependencyInjection\Extension` class, use the parent `Symfony\Component\DependencyInjection\Extension\Extension` instead

```diff
- use Symfony\Component\HttpKernel\DependencyInjection\Extension;
+ use Symfony\Component\DependencyInjection\Extension\Extension;

class ExampleExtension extends Extension
{
    // ...
}
```
````